src/main/java/uk/gov/hmcts/reform/jps/model/out/SittingRecord.java

```diff
@@ -46,6 +57,23 @@ public class SittingRecord {
     @ToString.Exclude
     private List<StatusHistory> statusHistories;
 
+    public String getCreatedByUserId() {
+        StatusHistory statusHistory = getFirstStatusHistory();
+        return null != statusHistory ? statusHistory.getChangeByUserId() : null;
+    }
+
+    @JsonIgnore
+    public StatusHistory getFirstStatusHistory() {
+        if (null == statusHistories) {
+            return null;
+        }
+
+        List<StatusHistory> statusHistoriesCopy = statusHistories.stream()
+            .sorted(Comparator.comparingLong(StatusHistory::getId))
+            .toList();
+        Optional<StatusHistory> optStatHistory = statusHistoriesCopy.stream().findFirst();
+        return optStatHistory.orElse(null);
+    }
 
     @Override
     public int hashCode() {
```

Comment:
Needed as we have our custom equals() method.
Otherwise following message:- 
warning: Not generating hashCode: One of equals or hashCode exists. You should either write both of these or none of these (in the latter case, lombok generates them).
@Data